docker/google-cloud-securitycenter/Dockerfile

```diff
@@ -0,0 +1,9 @@
+
+FROM demisto/python3:3.8.6.14516
+
+COPY requirements.txt .
+
+RUN apk add --no-cache libstdc++
```

Comment:
Please add --update. To make sure we install the  latest and greatest.

Reply:
Thank you @glicht for the review comment. We have added --update for libstdc++ library.